### ISSUE TITLE
Clear tag in kits removes flag item from head slot

### ISSUE THREAD

**Issue by RuedigerLP** [PGMDev/PGM#980]
Whn carrying the flag and applying a kit witha clear tag the banner that represents the flag being carried gets removed.

**Comment by Electroid** [comment]
@Pablete1234 What do you think about this issue? Should we detect when the helmet is set when carrying the flag and defer the application until the player drops the flag?

**Comment by Pablete1234** [comment]
Honestly i'm unsure if this is just a "works-as-intended" or not, potentially if you're holding a flag the item that gets cleared with the <clear> should be the "hidden/saved" helmet slot (the item stored in the flag state), but at the same time it should act as if you're dropping the flag

Does forcing a drop of the flag (by clearing the player's head slot) have any practical uses? If the answer is no, then it should just clear the item inside the flag state, if the answer is yes, maybe it should force dropping the flag and then clearing all inventory (including the helmet)

**Comment by RuedigerLP** [comment]
Yes, force dropping a flag would have uses, mostly in arcady maps.
